docs/docs/hpc-on-k8s/pytorch.md

```diff
@@ -0,0 +1,87 @@
+# PyTorch Workload
+PyTorch is an open-source machine learning library based on the Torch library, used for natural language processing (NLP), computer vision, and other artificial intelligence (AI) tasks.
+
+
+## Steps to Execute Job
+Below is the example manifest file for PyTorch workload in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: kubeflow.org/v1
+kind: PyTorchJob
+metadata:
+  name: pytorch-simple
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: shubhamdang/my_pytorch_image:latest
```

Comment:
Is this still the image repository or has the openflight one been created now?

Reply:
updated and image is present in openflighthpc dockerhub account.